### ISSUE TITLE
Training on my dataset

### ISSUE THREAD

**Issue by siyazingca** [yunjey/StarGAN#3]
I wanted to know if is it possible to train my dataset, I want to convert eyes from opened to closed and also the mouth

**Comment by yunjey** [comment]
@siyazingca Does the image contain one attribute (e.g., one-hot vector label)? Then, you can train the model in the same way as training on RaFD.

```
$ python main.py --mode='train' --dataset='RaFD' --c_dim='dimension of  one-hot vector' --image_size='image resolution you want' \
                 --num_epochs='number of epochs' --num_epochs_decay='number of epochs to decay' --sample_step='number of iteration for sampling' --model_save_step='number of iteration for saving model checkpoints' \
                 --sample_path='stargan_your_dataset/samples' --log_path='stargan_your_dataset/logs' \
                 --model_save_path='stargan_your_dataset/models' --result_path='stargan_your_dataset/results'
```

**Comment by happsky** [comment]
@yunjey what is one-hot vector label like, can you provide an example?

**Comment by happsky** [comment]
@siyazingca have you figure out how to train on yourself datasets?

**Comment by happsky** [comment]
@yunjey your work is a good baseline, can you provide more information about preparing the one-hot vector label on my own datasets?

**Comment by happsky** [comment]
@yunjey thanks for you sharing your work and this is a good work, can you provide more information about how to prepare the one-hot vector label on RaFD?

**Comment by yunjey** [comment]
@happsky You don't need to provide the one-hot vector label manually. Only you need to do is to create your data folder structure as described [here](https://github.com/yunjey/StarGAN/blob/master/png/RaFD.md). Then you can use the command below.

```
$ python main.py --mode='train' --dataset='RaFD' --c_dim=8 --image_size=128 \
                 --num_epochs=200 --num_epochs_decay=100 --sample_step=200 --model_save_step=200 \
                 --sample_path='stargan_rafd/samples' --log_path='stargan_rafd/logs' \
                 --model_save_path='stargan_rafd/models' --result_path='stargan_rafd/results'
```
